pipeline_tools/input_utils.py

```diff
@@ -21,6 +21,27 @@ def get_sample_id(bundle):
     return sample_id
 
 
+def get_expected_cell_count(bundle):
+    """Return the total estimated cells from the given bundle, otherwise use the same default value as CellRanger
+    (3000 cells): https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/count
+
+    Args:
+        bundle (humancellatlas.data.metadata.Bundle): A Bundle object contains all of the necessary information.
+
+    Returns:
+        total_estimated_cells (str): Int giving the total number of estimated cells
```

Comment:
nit: `total_estimated_cells (int): Int giving the total number of estimated cells`